packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/shouldSimplifySmalltalkCiLog.st

```diff
@@ -0,0 +1,4 @@
+accessing
+shouldSimplifySmalltalkCiLog
+	
+	^ shouldSimplifySmalltalkCiLog ifNil: [ shouldSimplifySmalltalkCiLog := false ]
```

Comment:
```suggestion
accessing
shouldSimplifyLog
	
	^ shouldSimplifyLog ifNil: [ shouldSimplifyLog := false ]
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code